pkg/webhook/cspc.go

```diff
@@ -341,9 +341,13 @@ var (
 	// TODO: confirm all the compression types supported by control plane
 	// and update the map accordingly
 	SupportedCompression = map[string]bool{
```

Comment:
Add a comment saying this can be relooked when user request for gzip-[1-9] compression support...

Reply:
added the comment